packages/neon-core/src/sc/NativeContractMethodPrices.ts

```diff
@@ -0,0 +1,27 @@
+import { Fixed8 } from "../u";
+
+export const NativeContractMethodPrices = new Map<string, Fixed8>([
+  ["getMaxTransactionsPerBlock", new Fixed8(1000000e-8)],
+  ["getFeePerByte", new Fixed8(1000000e-8)],
+  ["getBlockedAccounts", new Fixed8(1000000e-8)],
+  ["setMaxTransactionsPerBlock", new Fixed8(3000000e-8)],
+  ["setFeePerByte", new Fixed8(3000000e-8)],
+  ["blockAccount", new Fixed8(3000000e-8)],
+  ["unblockAccount", new Fixed8(3000000e-8)],
+  ["supportedStandards", new Fixed8(0)],
+  ["name", new Fixed8(0)],
+  ["symbol", new Fixed8(0)],
+  ["decimals", new Fixed8(0)],
+  ["totalSupply", new Fixed8(1000000e-8)],
```

Comment:
all these seem to be the NEP5 token contract. Is there going to be only 1 native contract forever? Or should we rename this to NativeTokenContractMethodPrices

Reply:
> Is there going to be only 1 native contract forever? 
I don't think I get what you mean, there are 3 native contracts now: NEO, GAS, POLICY, there would possibly be more in future. These contracts have some same-name methods, and their prices are same. That's why I use this to map price to method but not to contract.
>  should we rename this to NativeTokenContractMethodPrices
I think not. Because POLICY contracts are not tokens.

Reply:
but this class suggests that i can call totalSupply, balanceOf and all the method on Policy contract since it is considered a native contract. Or is Policy contract not native?

Reply:
And why are we adding sc within a tx, wallet PR

Reply:
Policy is native.
I see what you mean. How about not exporting this class from `sc` module?
> And why are we adding sc within a tx, wallet PR
Because in tx has methods to calculate systemFee and it needs the support from `sc`